src/main/java/org/asamk/Signal.java

```diff
@@ -81,7 +81,9 @@ long sendGroupMessageReaction(
             byte[] groupId, String name, List<String> members, String avatar
     ) throws Error.AttachmentInvalid, Error.Failure, Error.InvalidNumber, Error.GroupNotFound, Error.InvalidGroupId;
 
-    boolean isRegistered();
```

Comment:
please keep the `isRegistered()` method for backwards compatibility.

Reply:
done